server/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@elm-tooling/elm-language-server",
+  "version": "1.6.3",
+  "dependencies": {
+    "@elm-tooling/elm-language-server": "1.6.3"
```

Comment:
It's generally a good practice to also have package-lock.json committed to lock all sub-dependencies. In theory, they shouldn't break anything if they change but who knows. And it's good for users to have exactly the same code as to avoid confusion when there is a problem and we have to debug it.